<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputTest</class>
 <widget class="QDialog" name="InputTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InputTest</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#InputTest QPushButton {
color: #333;
border: 2px solid #555;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #888);
min-width: 80px;
}

#InputTest QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}

#InputTest QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}
#InputTest QPushButton:disabled {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
color: red;
}

#InputTest {
border: none;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #a6a6a6, stop: 0.08 #7f7f7f,
stop: 0.39999 #717171, stop: 0.4 #626262,
stop: 0.9 #4c4c4c, stop: 1 #333333);
}

#InputTest QLabel {
	color: rgb(255,242, 0);

}
QMessageBox{
	background-color: black;
	color: #fff200;
}</string>
  </property>
  <widget class="QPushButton" name="btStart">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>320</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>执行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btExit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>530</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btClear">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>80</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>命令</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btOpenSerial">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>250</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>读串口</string>
   </property>
  </widget>
  <widget class="QComboBox" name="serialPortInfoListBox">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 CN Regular</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>串口：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="baudRateBox">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>210</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>210</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 CN Regular</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>波特率：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btStopSerial">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>320</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>断开串口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btViewer">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>530</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>view serial</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>441</width>
     <height>131</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;at^sysinfo&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btCsq">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>94</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>信号质量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt2g">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>2G 模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt3g">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>3G模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btInfo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>470</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>网络信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btInit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>470</y>
     <width>94</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>初始化</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btReadcfg">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>读模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btUniwap">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>470</y>
     <width>94</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>3gwap</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btUninet">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>470</y>
     <width>94</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>3gnet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btAt">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>470</y>
     <width>94</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>AT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btNum">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>470</y>
     <width>94</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>num</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btBase">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>390</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>基站</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btcimi">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CIMI</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNum">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>540</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>手机号码::</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btSysinfo">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>530</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>sysinfo</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
